docs/user_guide.rst

```diff
@@ -95,6 +95,13 @@ quantum computer, and use Qiskit's ``execute`` function as usual:
 
     print(job.result().get_counts())
 
+.. note::
+
+   Qiskit's :meth:`qiskit.execute` method performs transpilation by default. If you want to inspect the transpiled
+   circuits, refer to `circuit_callback` option in the execution options table below. If you want to execute circuits
+   without automatic transpilation, you can use the :meth:`.IQMBackend.run` method directly; in that case you have to
+   take care of transpilation yourself.
```

Comment:
Should we mention`transpile_to_IQM` can be used for doing the default transpiration manually?

Reply:
Added.